sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -330,7 +330,7 @@ namespace Azure { namespace Core {
     Context WithValue(const std::string& key, ContextValue&& value) const
     {
       return Context{std::make_shared<ContextSharedState>(
```

Comment:
@CaseyCarter, you run clang format on this file to fix formatting.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=746260&view=logs&j=0585f5d6-0937-5c15-894c-06e189e06847&t=a45688da-a69e-5817-f832-3aa6246959b7&l=24
```text
Some files were not formatted correctly according to the .clang-format file.
Please run clang-format version 10 or greater to fix the issue by using this bash command at the root of the repo:
find ./sdk \( -iname '*.hpp' -o -iname '*.cpp' \) ! -iname 'json.hpp' -exec clang-format -i {} \;

List of files not formatted correctly:
sdk/core/azure-core/inc/azure/core/context.hpp

```

Reply:
Formatting + tests: https://github.com/CaseyCarter/azure-sdk-for-cpp/pull/1

Reply:
Resolved in e911b2d.